### ISSUE TITLE
The `mgz_pagebuilder` tags should not be shown in the dropdown with a search results

### ISSUE THREAD

**Issue by dmitrii-fediuk** [vegandthecity/magento#25]
![03](https://user-images.githubusercontent.com/83204/80264409-e35dcf80-86b5-11ea-9599-19c15a123daa.png)

**Comment by dmitrii-fediuk** [comment]
It is because the «[Magezon Page Builder](https://www.magezon.com/magezon-page-builder-for-magento-2.html)» (`Magezon_Builder`) module stores values of the `short_description` product attribute in a custom format:

![02](https://user-images.githubusercontent.com/83204/80267955-93870480-86c5-11ea-97ff-6dedeae4f28c.png)

![03](https://user-images.githubusercontent.com/83204/80267963-984bb880-86c5-11ea-8413-f62a31c47cec.png)

A value:
```json
[mgz_pagebuilder]{"elements":[{"row_type":"full_width_row","content_position":"top","device_type":"all","background_type":"image","background_style":"auto","background_position":"center-top","parallax_speed":0.5,"mouse_parallax_size":30,"mouse_parallax_speed":10000,"lg_background_type":"image","lg_background_style":"auto","lg_background_position":"center-top","md_background_type":"image","md_background_style":"auto","md_background_position":"center-top","sm_background_type":"image","sm_background_style":"auto","sm_background_position":"center-top","xs_background_type":"image","xs_background_style":"auto","xs_background_position":"center-top","id":"j1k6j1nkl3ey","type":"row","control":false,"hover":false,"hovered":false,"loading":false,"elements":[{"device_type":"all","background_type":"image","background_style":"auto","background_position":"center-top","parallax_speed":0.5,"mouse_parallax_size":30,"mouse_parallax_speed":10000,"lg_background_type":"image","lg_background_style":"auto","lg_background_position":"center-top","md_background_type":"image","md_background_style":"auto","md_background_position":"center-top","sm_background_type":"image","sm_background_style":"auto","sm_background_position":"center-top","xs_background_type":"image","xs_background_style":"auto","xs_background_position":"center-top","md_size":"","xl_offset_size":"","xl_size":"","lg_offset_size":"","lg_size":"","md_offset_size":"","sm_offset_size":"","sm_size":"","xs_size":"","id":"jl13pa3yesq5","type":"column","control":false,"hover":false,"hovered":false,"loading":false,"elements":[{"content":"<p><span>Mit dieser ökologischen Holz-Etikette zum Stecken vergisst du nicht mehr was du gesät und gepflanzt hast.</span></p>","device_type":"all","background_type":"image","background_style":"auto","background_position":"center-top","parallax_speed":0.5,"mouse_parallax_size":30,"mouse_parallax_speed":10000,"lg_background_type":"image","lg_background_style":"auto","lg_background_position":"center-top","md_background_type":"image","md_background_style":"auto","md_background_position":"center-top","sm_background_type":"image","sm_background_style":"auto","sm_background_position":"center-top","xs_background_type":"image","xs_background_style":"auto","xs_background_position":"center-top","id":"fe76asik5ffw","type":"text","control":false,"hover":false,"hovered":false,"loading":false,"elements":[]}]}]}]}[/mgz_pagebuilder]
```

![01](https://user-images.githubusercontent.com/83204/80267975-b31e2d00-86c5-11ea-980a-5b75dd2b5141.png)

**Comment by dmitrii-fediuk** [comment]
Fixed in [0.0.7](https://github.com/vegandthecity/core/releases/tag/0.0.7).
